### DISCUSSION TITLE
Welcome to OMaps Discussions!

### DISCUSSION THREAD

**Discussion by rtsisyk** [organicmaps/organicmaps#58] [Q&A]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by FinixFighter** [comment]
Hi! I'm an OpenStreetMap contributor and Organic Maps is my favourite app for routing and searching places. I have many passions, one of them is free and open source software!
